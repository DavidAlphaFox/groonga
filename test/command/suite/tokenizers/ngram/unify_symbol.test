tokenize \
  'TokenNgram("unify_symbol", false)' \
  "___---" \
  NormalizerAuto
