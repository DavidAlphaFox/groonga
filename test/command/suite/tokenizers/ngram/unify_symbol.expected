tokenize   'TokenNgram("unify_symbol", false)'   "___---"   NormalizerAuto
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "__",
      "position": 0,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "__",
      "position": 1,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "_-",
      "position": 2,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "--",
      "position": 3,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "--",
      "position": 4,
      "force_prefix": false,
      "force_prefix_search": false
    },
    {
      "value": "-",
      "position": 5,
      "force_prefix": false,
      "force_prefix_search": false
    }
  ]
]
